src/components/Header/UnderHeader/under_header.tsx

```diff
@@ -1,9 +1,12 @@
-import React from "react";
-import { useSelector } from "react-redux";
+import React, { useState } from "react";
+import { useDispatch, useSelector } from "react-redux";
+import { getJobsFilter } from "../../../redux/jobs/jobs-action";
 import { RootStore } from "../../../redux/store";
 
 const UnderHeader: React.FC = () => {
+  const dispatch = useDispatch();
   const jobState = useSelector((state: RootStore) => state.job);
+  const [inputSearch, SetInputSearch] = useState<string>("");
```

Comment:
Why did you upper first character of `SetInputSearch`? It should be `setInputSearch`.

Reply:
sorry Sir, my fatal mistake